setup.py

```diff
@@ -304,6 +304,8 @@ def init_settings_from_config(self):
             pjoin('backend', 'cython'),
             'devices',
         )]
+        if sys.platform.startswith('win'):
+            settings['include_dirs'].insert(0, 'patch')
```

Comment:
When I remove it, I get this error:

    monitoredqueue.c
    bundled\zeromq\include\zmq.h(78) : fatal error C1083: Cannot open include file: 'stdint.h': No such file or directory

The `stdint.hpp` it provides is used only internally.  I guess `libzmq` also depends on `<stdint.h>` being available to client applications...